### DISCUSSION TITLE
Dozzle logs slow to load when using socket-proxy

### DISCUSSION THREAD

**Discussion by robflate** [htpcBeginner/docker-traefik#108] [Support]
If I connect Dozzle directly to the docker socket, the list of containers/logs load instantly when I visit `dozzle.domain.com`;
```
volumes:
      - /var/run/docker.sock:/var/run/docker.sock
```
However, if I use the socket-proxy, the list of containers/logs load much slower (10 to 12 seconds)
```
environment:
      DOCKER_HOST: tcp://socket-proxy:2375
```
The Dozzle GitHub mentions [slow loading when using a proxy](https://github.com/amir20/dozzle#troubleshooting-and-faqs) in the Troubleshooting and FAQs section, specifically referring to buffers but I'm not sure how to implement the fix. Is anyone else experiencing this?

**Comment by mbelangergit** [comment]
I see this behavior on my setup too. Still looking for a fix…

It may still related, but I think the FAQ you are referring is on issues with reverse proxies and not socket-proxy.

**Comment by SimpleHomelab** [comment]
It loads slowly for me as well but it hasnt bothered me. 

This is not an issue with this repo. Therefore, moving to discussions.

**Reply by shawnthompson** [reply]
@htpcBeginner you can move issues directly to discussions? How cool is that?

**Reply by SimpleHomelab** [reply]
It is. Thanks to you @shawnthompson for asking for a community. Otherwise I would not have looked into it. Also thanks for being active here.

**Reply by shawnthompson** [reply]
Thanks for publishing your guides.
